examples/pulse_control/1_Bringup_experiments.ipynb

```diff
@@ -11,7 +11,7 @@
     "- calibrate a $\\pi$/2 pulse via Rabi spectroscopy\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print("Qauntum Task Summary")
</pre><p>If we are revising this, can we clean up the spelling here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/683/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>